common/src/main/java/org/jboss/hal/testsuite/creaper/ResourceVerifier.java

```diff
@@ -236,6 +236,44 @@ public ResourceVerifier verifyAttributeIsUndefined(String attributeName) throws
         return verifyAttributeIsUndefined(attributeName, null);
     }
 
+    /**
+     * Verifies that list type attribute contains give value.
+     * @param value Value which should be present in the list.
+     */
+    public ResourceVerifier verifyListAttributeContainsValue(String attributeName, ModelNode value, String errorMessageSuffix) throws Exception {
+        waitFor(() -> {
+            ModelNodeResult actualResult = ops.readAttribute(resourceAddress, attributeName);
+            return actualResult.isSuccess() && actualResult.hasDefined(Constants.RESULT);
```

Comment:
Is it sufficient to just verify that the attribute is readable and not that expected list item is already propagated to it?